stdlib/public/Platform/android.modulemap

```diff
@@ -0,0 +1,661 @@
+  // and networking.
+  module ifaddrs {
+    header "ifaddrs.h"
+    export *
+  }
+  module netdb {
+    header "netdb.h"
+    export *
+  }
+  module arpa_inet {
+    header "arpa/inet.h"
+    export *
+  }
+  module netinet_in {
+    header "netinet/in.h"
+    export *
+  }
+  module netinet_in6 {
+    header "netinet/in6.h"
+    export *
+  }
+  export *
+}
+
+module _jni [system] {
+  header "jni.h"
+  export *
+}
+
+module _langinfo [system] {
+  header "langinfo.h"
+  export *
+}
+
+module _pthread [system] {
+  header "pthread.h"
+  export *
+}
+
+module _regex [system] {
+  header "regex.h"
+  export *
+}
+
+module _resolv [system] {
+  header "resolv.h"
+  export *
+}
+
+module _sched [system] {
+  header "sched.h"
+  export *
+}
+
+module _search [system] {
+  header "search.h"
+  export *
+}
+
+module _semaphore [system] {
+  header "semaphore.h"
+  export *
+}
+
+module _spawn [system] {
+  header "spawn.h"
+  export *
+}
+
+module _strings [system] {
+  header "strings.h"
+  export *
+}
+
+module _sys [system] {
+  module syscall {
+    header "syscall.h"
+    export *
+  }
+  module sysexits {
+    header "sysexits.h"
+    export *
+  }
+  module syslog {
+    header "syslog.h"
+    export *
+  }
+  export *
+}
+
+module _tar [system] {
+  header "tar.h"
+  export *
+}
+
+module _termio [system] {
+  module termio {
+    header "termio.h"
+    export *
+  }
+  module termios {
+    header "termios.h"
+    export *
+  }
+  export *
+}
+
+module _threads [system] {
+  header "threads.h"
+  export *
+  explicit module threads_inlines {
+    header "bits/threads_inlines.h"
+    export *
+  }
+}
+
+module _uconfig_local [system] {
+  header "uconfig_local.h"
+  export *
+}
+
+module _ucontext [system] {
+  header "ucontext.h"
+  export *
+}
+
+module _wait [system] {
+  header "wait.h"
+  export *
+  explicit module sys_wait {
+    header "sys/wait.h"
+    export *
+  }
+}
+
+module _wchar_uchar [system] {
+  module uchar {
+    header "uchar.h"
+    export *
+  }
+  module wchar {
+    header "wchar.h"
+    export *
+  }
+  explicit module mbstate_t {
+    header "bits/mbstate_t.h"
+    export *
+  }
+  export *
+}
+
+module _xlocale [system] {
+  header "xlocale.h"
+  export *
+}
+
+// Additional modules in the 'android' subdirectory.
+module _android_defs [system] {
+  module ndk_version {
+    header "android/ndk-version.h"
+    export *
+  }
+  module versioning {
+    header "android/versioning.h"
+    export *
+  }
+  export *
+}
+
+module _android_apis [system] {
+  module asset_manager_jni {
+    header "android/asset_manager_jni.h"
+    export *
+  }
+  module asset_manager {
+    header "android/asset_manager.h"
+    export *
+  }
+  module log {
+    header "android/log.h"
+    export *
+  }
+  module trace {
+    header "android/trace.h"
+    export *
+  }
+  export *
+}
+
+// Additional modules in the 'bits' subdirectory.
+module _bits_io_inet [system] {
+  module in_addr {
+    header "bits/in_addr.h"
```

Comment:
Should some/all of these be in the `inet` module? They seem to go with those headers

Reply:
Yup moved them over , except the sa_address to prevent circular conflict. I added a comment noting why it's separate.